pymeasure/instruments/rohdeschwarz/fsl.py

```diff
@@ -302,15 +302,31 @@ def select_channel(self, channel_name):
         """
         self.write(f"INST:SEL '{channel_name}'")
 
-    # active_channel = Instrument.control("INST?", "INST %s", "To get the active channel")
+    @property
+    def active_channel(self):
```

Comment:
This should get a docstring.

Reply:
Why are you adding underscores?

Also again, please have a look at docstring style to be used. It is also outlines in the pymeasure documentation: https://pymeasure.readthedocs.io/en/latest/dev/coding_standards.html#docstrings

Reply:
Resolved via [9d73b1d](https://github.com/bleykauf/pymeasure/pull/4/commits/9d73b1d02c1df0a713dcb7e6d7fef6921f07e2e9)